### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Did some minor code changes
- Added search bar on the quiz category 

## Preview Link:
https://deploy-preview-6--guessthisquiz.netlify.app/

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -16,6 +16,23 @@ const Navbar = () => {
...
+            name="search"
+            id="search"
+            placeholder="🔍 Search"
+            // onChange={(e) =>
+            //   filterDispatch({
+            //     type: "FILTER_BY_SEARCH",
+            //     payload: e.target.value,
+            //   })
+            // }
+            // value={filterState.searchKeyword}
```
Comment (rohanmathur91) [inline src/components/Navbar/Navbar.jsx:31]: remove this if not needed

**Review by rohanmathur91 — APPROVED** [review]
- Good folder structure and clean coding
- Nice naming of components
- Overall looks good, great work!

File: src/reducer/currentQuestion-reducer.js [hunk 2]
```diff
@@ -11,6 +11,15 @@ const currentQuestionReducer = (state, action) => {
           action.payload.selectedOption === action.payload.correctAnswer
             ? state.score + 10
             : state.score,
+        results: [
+          ...state.results,
+          {
+            currentQuestion: action.payload.currentQuestion,
```
Comment (Rohitprasad83) [inline src/reducer/currentQuestion-reducer.js:17]: can destructure the action.payload

**Review by Rohitprasad83 — APPROVED** [review]
Nice code structure, just added a suggestion. Good to merge

**Pull request merged**